ci-operator/step-registry/ipi/conf/customnoupgrade/ipi-conf-customnoupgrade-ref.yaml

```diff
@@ -0,0 +1,15 @@
+ref:
+  as: ipi-conf-customnoupgrade
+  commands: ipi-conf-customnoupgrade-commands.sh
+  from: cli
+  resources:
+    requests:
+      cpu: 10m
+      memory: 100Mi
+  env:
+    - name: CUSTOM_FEATURE_FLAGS_ENABLED
+      default: ""
+    - name: CUSTOM_FEATURE_FLAGS_DISABLED
+      default: ""
```

Comment:
Please add a `documentation` stanza specifying the required format. Which brings me to: Why don't you just directly accept a comma-separated list?

Reply:
I worried about consistency so I did a quick look at other uses of lists in releases/. Here are the first 3 I looked at:

`AZURE_ARC_REGIONS`: space separated
`ADDON_PARAMETERS`: json string
`CLUSTERPOOL_LIST`: comma separated list

I stopped looking at this point. I conclude there is no consistency and I'm free to specify a comma separated list here.